pages/faq.tsx

```diff
@@ -219,7 +219,48 @@ const questions: ReadonlyArray<{
     question:
       'I get the error, "Your database needs to be upgraded (v10 vs v14)."',
     id: 'database-error',
-    answer: `You are running phase 3 on an old phase 2 database. You need to delete your datadir located at ~/.ironfish`,
+    answer: (
+      <div>
+        You are running phase 3 on an old phase 2 database. You need to delete
+        your old datadir. Please first try the command{' '}
+        <code>ironfish reset</code>. If that does not work try{' '}
+        <code>rm -rf ~/.ironfish</code>
+      </div>
+    ),
+  },
+  {
+    question: 'I get a "SIGSEGV" error when running my node',
+    id: 'sigsegv',
+    answer: (
+      <div>
+        We are aware of this issue, please submit your crash report{' '}
+        <span className="underline">
+          <Link href="https://github.com/iron-fish/ironfish/issues/2985">
+            here
+          </Link>
+        </span>{' '}
+        and follow along for more details.
+      </div>
+    ),
+  },
```

Comment:
I don't think anyone is gonna look at the FAQ for this, it's gonna be better in a "known issues list". I also don't like to put temporary things into the FAQ because as soon as we fix this we need to come in here and remove this.